### PR TITLE
Generate APIDoc on dockerfile

### PR DESCRIPTION
**Pull request by biancarosa**
# Description

Generate APIDoc on dockerfile

Fixes part of #THEEDGE-1507

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `go fmt ./...` to check that my code is properly formatted
- [x] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,8 +49,6 @@ RUN rm -rf /mnt/rootfs/var/cache/* /mnt/rootfs/var/log/dnf* /mnt/rootfs/var/log/
 FROM scratch
 LABEL maintainer="Red Hat, Inc."
 LABEL com.redhat.component="ubi8-micro-container"
-
-# label for EULA
```
Comment (Avielyo10) [inline Dockerfile:50]: I suggest to keep this comment so no one will remove the label underneath it which is super important (in order for us to investigate the image for vulnerabilities

**Comment by Avielyo10** [comment]
You may wanna change travis spec validation to run from the container itself. And also change the github action to build the spec binary then test it.

This way we test the container validation using travis & the binary validation using github action (thou the dockerfile won't build if the binary is broken - that's totally cool right now)

**Comment by biancarosa** [comment]
Hi @Avielyo10! Thanks for the suggestion, I was trying to figure out how do it since I didn't know it was going to break. Gonna work on that!

**Comment by biancarosa** [comment]
@Avielyo10 ended up commenting travis for now and I was thinking of opening a second PR for that as we need python on the test container. Thoughts?

**Comment by maxamillion** [comment]
@biancarosa  +1 split to another PR

**Review by maxamillion — APPROVED** [review]

**Pull request merged**